From: Adam Gandelman <adamg@ubuntu.com>
Date: Thu May 23 19:12:03 PDT 2013
Remove version requirements of paramiko dependency until new upstream paramiko
version is packaged.  Current version in Saucy (1.7.7.1-3.1ubuntu1) has a
cherry-picked patch to avoid the bug that prompted the version requirement
here.


diff -Naurp cinder-2013.2.b1.orig/tools/pip-requires cinder-2013.2.b1/tools/pip-requires
--- cinder-2013.2.b1.orig/tools/pip-requires	2013-06-19 12:37:49.147728578 -0500
+++ cinder-2013.2.b1/tools/pip-requires	2013-06-19 12:41:13.171728677 -0500
@@ -11,7 +11,7 @@ kombu>=1.0.4
 lockfile>=0.8
 lxml>=2.3
 oslo.config>=1.1.0
-paramiko>=1.8.0
+paramiko
 paste
 pastedeploy>=1.5.0
 python-glanceclient>=0.5.0,<2
